### PR TITLE
Add documentation to the backend API (v2)

### PR DESCRIPTION
**Pull request by TeamoreA**
### What does this PR do?
Adds descriptions to the Graphql queries and mutations.
### Description of the task to be completed?
When one visits the documentation they should be able to see the respective descriptions of both the mutations and the queries.
### How should this be manually tested?
 - Clone the branch and follow the following steps to [setup](https://github.com/andela/mrm_api).
 - Checkout by running `git checkout ft-documentation-v2-164135004`
 - Go to the docs tab on the top right of the browser to see the descriptions of various queries and mutations.

### Any background context you want to provide?
N/A
### What are the relevant pivotal tracker stories?
[#164135004](https://www.pivotaltracker.com/story/show/164135004)
### Screenshots
 - #### Description of a mutation that creates a floor
image

- #### Description of a query to get a room
image

### CODE REVIEW

File: api/room/schema.py [hunk 1]
```diff
@@ -193,7 +238,37 @@ def mutate(self, info, room_id, **kwargs):
...
+            \n- name: The name of the room[required]\
+            \n- room_type: The type of the given room\
+            \n- capacity: The number of peaople that the room can hold\
+            [required]\n- image_url: The image url of the room\
+            \n- location_id: The unique identifier of the location where the \
+            room is found\n- floor_id: The unique identifier of the floor where\
+            the room is found[required]\n- calendar_id: The unique identifier \
+            of the calendar\n- office_id: The unique identifier of the office \
+            where the room is found[required]\n- wing_id: The unique identifier\
+            of the wing where the room is found\n- block_id: The unique \
```
Comment (Bonifase) [inline api/room/schema.py]: I don't think we still need block_id and wing_id here

File: api/location/schema.py [hunk 2]
```diff
@@ -97,6 +106,9 @@ def mutate(self, info, location_id, **kwargs):
 
 
 class DeleteLocation(graphene.Mutation):
+    """
```
Comment (RachaelNantale) [inline api/location/schema.py]: Please provide more descriptive documentation. This is misleading

File: api/office/schema.py [hunk 3]
```diff
@@ -19,11 +19,17 @@
 
 
 class Office(SQLAlchemyObjectType):
+    """
```
Comment (dnuwa) [inline api/office/schema.py]: @TeamoreA this is good work though I am curious why we still have the `office/schema.py` in the v2 codebase

Comment (TeamoreA) [reply api/office/schema.py]: It's already in v2. I got it after branching off v2

**Pull request merged**